docs/build/smart-contracts/getting-started/hello-world.mdx

```diff
@@ -150,14 +151,14 @@ All contracts should begin with `#![no_std]` to ensure that the Rust standard li
 The contract imports the types and macros that it needs from the `soroban-sdk` crate.
 
 ```rust
-use soroban_sdk::{contract, contractimpl, symbol_short, vec, Env, Symbol, Vec};
+use soroban_sdk::{contract, contractimpl, vec, Env, String, Vec};
 ```
 
 Many of the types available in typical Rust programs, such as `std::vec::Vec`, are not available, as there is no allocator and no heap memory in Soroban contracts. The `soroban-sdk` provides a variety of types like `Vec`, `Map`, `Bytes`, `BytesN`, `Symbol`, that all utilize the Soroban environment's memory and native capabilities. Primitive values like `u128`, `i128`, `u64`, `i64`, `u32`, `i32`, and `bool` can also be used. Floats and floating point math are not supported.
 
 Contract inputs must not be references.
 
-The `#[contract]` attribute designates the Contract struct as the type to which contract functions are associated. This implies that the struct will have contract functions implemented for it.
+The `#[contract]` attribute designates the `HelloContract` struct as the type to which contract functions are associated. This implies that the struct will have contract functions implemented for it.
```

Comment:
~~The CLI (at least `v22.2.0`, which I _think_ is the current version) creates a struct called `Contract`.~~

Edit: Never mind, we're talking about the astro template, not the `stellar contract init` command... 🤦🏻‍♂️ 

Edit Edit: Oh, wait, no... This page isn't talking about the astro template, right? We might need to modify instances of `HelloContract` to just `Contract` then?

Reply:
Good catch! You're right, the current version of the CLI just has `Contract`. 

I have a branch to update the code that `stellar contract init` creates to make it `HelloContract` so that it matches the soroban-examples repo, but I'll revert this back to `Contract` for now. And we can make this change when the cli change is also merged. I can open 2 new PRs.

Reply:
For reference, here are the 2 new Prs: https://github.com/stellar/stellar-docs/pull/1239 and https://github.com/stellar/stellar-docs/pull/1185